### PR TITLE
Works and About me Sections added

### PR DESCRIPTION
**Pull request by BregornOriginal**
Today I was working on using Flexbox and CSS Grid to place elements on the page.😄

# The mobile website section is added whit the next features
## In works sections were added:
- A image whit recent works and the technologies used
- A button in every project that will let you see the project (in-progress)
In the section "About me":
- There is a brief description "about me"
- A button to get My Resume
- And next the languages, Frameworks, and skills learned for now.
- The buttons have the feature :hover 

### Also added the files corresponding necessary

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,5 +59,232 @@ <h3>I'am a Software Developer</h3>
...
+            id="image-recent-works"
+            src="./images/image-section-one.png"
+            alt="A woman doing Yoga"
+          />
+        </div>
+        <div class="information-container">
+          <h5 class="title-recent-works-one">Multi-Post Stories</h5>
+          <p class="description-recent-works-one">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard dummy text ever since the 1500s, when an unknown printer
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly wrap your work grid inside a `section tag`

File: index.html [hunk 2]
```diff
@@ -59,5 +59,232 @@ <h3>I'am a Software Developer</h3>
...
+          <div>
+            <ul class="technologies-container-works-two">
+              <li class="technologies-works-two">HTML</li>
+              <li class="technologies-works-two">Bootstrap</li>
+              <li class="technologies-works-two">Ruby</li>
+            </ul>
+          </div>
+        </div>
+        <div class="button-container">
+          <button type="button" class="button-see-project-works-two">See Project</button>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly use a `div` tag for each project card

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @BregornOriginal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vic778 — APPROVED** [review]
Hi @BregornOriginal   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by BregornOriginal** [comment]
Hi @aimemalaika @vic778 ! 
Thank you for the reviews! Was good to take them so fast and it's were very clear!
Best!
Julio.

**Comment by vic778** [comment]
hi @BregornOriginal we're here to help each other.. happy coding and enjoy your codes :smile:

**Pull request merged**